docs/supported-technologies.asciidoc

```diff
@@ -7,7 +7,7 @@
 
 The RUM agent tries to capture the overall user experience on how the page is loaded/rendered as part of the page load Transaction.
 It includes all of the dependent resources, like JavaScript, stylesheets, images, etc., which are included as part of the page. In addition
-to capturing the resource timing information as Spans, the transaction also includes navigation spans that are associated with the rest
+to capturing the resource timing information as Spans, the transaction also includes navigation spans—marks that are associated with the rest
 of the captured information to make the waterfall more appealing.
```

Comment:
```suggestion
to capturing the resource timing information as Spans, the transaction also includes navigation marks that annotate important events on the transaction waterfall.
```

Reply:
We want to showcase both navigation spans and marks, The suggestion only takes marks in to account which is not correct. going to keep the previous version.

Reply:
I think we need to rewrite this paragraph, IMO it's not clear what exactly is the functionality that is described. I think part of that is because we're trying to combining the navigation timing spans with the marks.

Reply:
@bmorelli25 Any ideas here?

Reply:
I am expanding this metrics section more on support and others, will tackle in that one. merging this PR as it is.